docs/Basics/01-working-with-projects.md

```diff
@@ -4,16 +4,58 @@ tags: []
 
 # Working with Projects
 
-![]()
+Projects in Studio provide a place for you to manage collections of APIs, articles, and any other files that you want to store together. You can have as many projects as you like, but keep in mind that you'll probably want to store similar files together when possible.
 
-## What 
-Already have an existing project or API Specification? Upload your API Specification or entire repository to Stoplight to begin working with your files immediately. 
+![Studio New Project Screen](../../assets/images/studio-project-screen.png)
 
-## How 
-1. Select the **Project Dropdown**
-2. Click **+ Project** 
-3. Input a **Name** for your Project 
-4. Click **Choose** under Local Directory 
-5. Select your API spec or Repo from your file explorer 
-6. Click **Open** 
+## Project Structure
 
+Studio projects assume a specific structure in order to group files efficiently as well as to not interfere with existing files if you are importing an existing repository. The structure is:
```

Comment:
link to the directory structure docs (documentation and modeling) for in-depth help with this